src/beautifiers/ocamlformat.coffee

```diff
@@ -28,10 +28,9 @@ module.exports = class OCamlFormat extends Beautifier
     OCaml: true
   }
 
-  beautify: (text, language, options) ->
-    @run("ocamlformat", [
-      @tempFile("input", text)
-      ], {
+  beautify: (text, language, options, context) ->
+    args = if context.filePath then ["--name", context.filePath] else []
```

Comment:
- [ ] The file extension `.mli` should be added here: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/ocaml.coffee#L16-L18

- [ ] Add an example to https://github.com/Glavin001/atom-beautify/tree/master/examples/nested-jsbeautifyrc/ocaml

Reply:
Done both but could you check if I got the second one right? Not sure how the formatting tests work and it looks like the OCaml ones are not run anyways.